test/rng_testsuite/statistics_tests/exponential_distribution_test.pass.cpp

```diff
@@ -0,0 +1,639 @@
+// -*- C++ -*-
+//===-- exponential_distribution_test.cpp ---------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Test of exponential_distribution - comparison with std::

```

Comment:
please, change the description, we do not compare results with std:: here, we check statistical properties of the distribution

Reply:
done